tests/cpu/ops/gradingprimary/GradingPrimary_tests.cpp

```diff
@@ -113,4 +121,34 @@ OCIO_ADD_TEST(GradingPrimary, precompute)
     OCIO_CHECK_ASSERT(comp.getBrightness() == OCIO::Float3({ -0.1f, 0.f, 0.f }));
     OCIO_CHECK_ASSERT(comp.getContrast() == OCIO::Float3({ 1.f, 0.8f, 1.f }));
     OCIO_CHECK_ASSERT(comp.getGamma() == OCIO::Float3({ 1.f, 1.f, 0.8f }));
+
+    // Test isGammaIdentity() / isContrastIdentity()
+    gp = OCIO::GradingPrimary{ OCIO::GRADING_LOG };
+
+    gp.m_gamma.m_red = 0.8;
+    comp.update(OCIO::GRADING_LOG, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(!comp.isGammaIdentity());
+    OCIO_CHECK_ASSERT(!comp.getLocalBypass());
+    gp.m_gamma.m_red = 1.0;
+    comp.update(OCIO::GRADING_LOG, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(comp.isGammaIdentity());
+    OCIO_CHECK_ASSERT(comp.getLocalBypass());
+
+    gp.m_contrast.m_red = 0.8;
+    comp.update(OCIO::GRADING_LIN, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(!comp.isContrastIdentity());
+    OCIO_CHECK_ASSERT(!comp.getLocalBypass());
+    gp.m_contrast.m_red = 1.0;
+    comp.update(OCIO::GRADING_LIN, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(comp.isContrastIdentity());
+    OCIO_CHECK_ASSERT(comp.getLocalBypass());
+
+    gp.m_gamma.m_red = 0.8;
+    comp.update(OCIO::GRADING_VIDEO, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(!comp.isGammaIdentity());
+    OCIO_CHECK_ASSERT(!comp.getLocalBypass());
+    gp.m_gamma.m_red = 1.0;
+    comp.update(OCIO::GRADING_VIDEO, OCIO::TRANSFORM_DIR_FORWARD, gp);
+    OCIO_CHECK_ASSERT(comp.isGammaIdentity());
+    OCIO_CHECK_ASSERT(comp.getLocalBypass());
```

Comment:
Minor:
It would be useful to have just a small comment about what these are testing exactly.

Reply:
I added a short description above tests, thanks for the suggestion.